ambari/user-guide/manage-hosts.rst

```diff
@@ -0,0 +1,230 @@
+
+.. image:: /img/ambari/hosts_filter_icons.png
+
+For example, to limit the list of hosts appearing on Hosts home to only those with Healthy status, select Filters, then choose the Healthy option. In this case, one host name appears on Hosts home. Alternatively, to limit the list of hosts appearing on Hosts home to only those having Maintenance Mode on, select Filters, then choose the Maintenance Mode option. In this case, three host names appear on Hosts home.
+
+Use the general filter tool to apply specific search and sort criteria that limits the list of hosts appearing on the Hosts page.
+
+Performing Host-Level Actions
+_____________________________
+
+Use Actions to act on one, or multiple hosts in your cluster. Actions performed on multiple hosts are also known as bulk operations.
+
+Actions comprises three menus that list the following option types:
+
+* Hosts - lists selected, filtered or all hosts options, based on your selections made using Hosts home and Filters.
+* Objects - lists component objects that match your host selection criteria.
+* Operations - lists all operations available for the component objects you selected.
+
+For example, to restart DataNodes on one host:
+
+1. In Hosts, select a host running at least one DataNode.
+2. In Actions, choose Selected ``Hosts > DataNodes > Restart``, as shown in the following image.
+
+  .. image:: /img/ambari/Host_datanode_restart.png
+
+3. Choose OK to confirm starting the selected operation.
+4. Optionally, use Monitoring Background Operations to follow, diagnose or troubleshoot the restart operation.
+
+Viewing Components on a Host
+____________________________
+
+To manage components running on a specific host, choose a FQDN on the Hosts page.To manage components running on a specific host, choose a FQDN on the Hosts page.
+
+.. image:: /img/ambari/ambari_host_detail.png
+
+Choose options in ``Host Actions``, to start, stop, restart, delete, or turn on maintenance mode for all components installed on the selected host.
+
+Alternatively, choose action options from the drop-down menu next to an individual component on a host. The drop-down menu shows current operation status for each component.
+
+Decommissioning Masters and Slaves
+__________________________________
+
+Decommissioning is a process that supports removing a component from the cluster. You must decommission a master or slave running on a host before removing the component or host from service. Decommissioning helps prevent potential loss of data or service disruption. Decommissioning is available for the following component types:
+
+* DataNodes
+* NodeManagers
+
+Decommissioning executes the following tasks:
+
+* For DataNodes, safely replicates the HDFS data to other DataNodes in the cluster.
+* For NodeManagers, stops accepting new job requests from the masters and stops the component.
```

Comment:
actually, decommission is menaged into ambari only for HDFS components.
decomission operations related to other farmework should be added in other serctions until ambari will not manage it